functorch/experimental/_nonzero_static.py

```diff
@@ -0,0 +1,65 @@
+import torch
```

Comment:
If you don't mind, I'd like to bikeshed where exactly we should put this operator for a bit.

It doesn't seem right to put random export-oriented experimental operators in functorch. Functorch is about function transformations, which is why higher order operators should live there; nonzero doesn't really fit the bill here.

At the same time, there is no obvious preexisting spot to put it. We currently define custom operators in torch/_prims/__init__.py and torch/ao/quantization/fx/_decomposed.py.

I think there's a decent case to be made that nonzero_static makes sense as a prim. What do you think about having it live in `torch/_prims/__init__.py`?

Reply:
Yeah, I don't feel it belongs to functorch, but no idea where is the best place for it. Since you suggested `torch/_prims/__init__.py`, a few questions about it:
 1. Are prim ops "final" ops that can not be further decomposed? If yes, and we want to put `nonzero_static` here, it means `nonzero_static` is not decomposable?
 2. No idea which type of op `nonzeor_static` belongs to, I currently put it under "Tensor Creation Prims"

Reply:
Yes, prim ops are final, not intended to be decomposed. It doesn't seem like you want to decompose nonzero_static anyway?

Tensor creation is not the right spot, but whatever, make a new section for it.